src/Altinn.Profile.Core/AltinnConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Altinn.Profile.Core
+{
+    /// <summary>
+    /// Configuration settings for Altinn Profile
+    /// </summary>
+    public class AltinnConfiguration
```

Comment:
I think we should try to come up with a more specific name. 

AddressMaintainanceSettings (or something?)